src/vendor/wordpress/_wordpress.scss

```diff
@@ -55,6 +83,40 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: Embed
+ * Applies our pattern library styles to gutenberg embeded blocks
+ */
+
+.wp-block-embed {
+  &.o-embed--square .wp-block-embed__wrapper {
+    --aspect-ratio: #{aspect-ratio.$square};
+  }
+
+  &.o-embed--full .wp-block-embed__wrapper {
+    --aspect-ratio: #{aspect-ratio.$full};
+  }
+
+  &.o-embed--cinema .wp-block-embed__wrapper {
+    --aspect-ratio: #{aspect-ratio.$cinema};
+  }
```

Comment:
Are these aspect ratios something that can be controlled via WordPress's interface, or will they require "hand-editing" of HTML?

Reply:
These aspect ratios can controlled by adding our existing `o-embed--` modifier classes to the "Advanced > additional CSS classes" input when an embed block is selected. 

While these are not controlled, by an easier, more intuitive interface that exposes the available modifier classes, I thought it would be worthwhile to account for their usage. Let me know if you disagree. 

<img width="1104" alt="Screen Shot 2020-08-19 at 10 26 31 AM" src="https://user-images.githubusercontent.com/1427548/90669520-a4c0c780-e206-11ea-9b0a-18e35f521c2f.png">

Reply:
@derekshirk I believe we can add support for responsive embeds, which would expose a UI for this so we don't have to manually add classes? https://developer.wordpress.org/block-editor/developers/themes/theme-support/#responsive-embedded-content